bin/chrootify

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
- Find a more suitable directory for this file. This probalby belongs in cwf/gateway/docker
- Document what this does because this is uncrackable right now. Decompose main appropriately into self-documenting helper functions to facilitate readability and maintainability. Add appropriate help and usage text to the arg parser